src/app/components/ExportModal.js

```diff
@@ -16,10 +16,11 @@ import { queryContactExport } from 'proton-shared/lib/api/contacts';
 import downloadFile from 'proton-shared/lib/helpers/downloadFile';
 import { wait } from 'proton-shared/lib/helpers/promise';
 
-import { bothUserKeys, prepareContact } from '../helpers/decrypt';
+import { bothUserKeys, prepareContact as decrypt } from '../helpers/decrypt';
```

Comment:
I don't like to rename function, it's a pain to maintain if we decide to change something related to this function. If you feel the name should be change, change it everywhere but don't do that please.

Reply:
Ok. It's not that necessary to change it here. There is one `prepareContact` in `decrypt.js` and another in `encrypt.js`, so in other places that I had to use both I had to do the renaming. I thought it was more readable like that, but indeed the renaming is not necessary here

Reply:
Thanks.